### PR TITLE
Remove `CommonDependencies`

### PR DESCRIPTION
**Pull request by jedesah**
- Upgrade a few dependencies like `pathy`, `rapture`, `refined`, `parser-combinators`, `aesh`, `h2`, `spark` `scala-xml` and a few others. Most are revision numbers so should be safe (not that tests wouldn't alert us of some breaking change)
- Add notes on attempt to upgrade `doobie`. Conclusion: wait for `0.4.2`

### CODE REVIEW

**Comment by jedesah** [comment]
Not the most convenient diff sorry...

File: project/Dependencies.scala [hunk 1]
```diff
@@ -63,20 +74,20 @@ object Dependencies {
...
-    CommonDependencies.doobie.core,
-    CommonDependencies.doobie.hikari,
-    CommonDependencies.doobie.postgres,
-    "org.http4s"           %% "http4s-core" % http4sVersion,
-    CommonDependencies.monocle.`macro`,
-    "com.github.tototoshi" %% "scala-csv"      % "1.3.4",
-    "com.slamdata"         %% "pathy-argonaut" % pathyVersion,
-    CommonDependencies.doobie.specs2                          % Test,
-    CommonDependencies.doobie.h2                              % Test
+    "org.tpolecat"               %% "doobie-core"     % doobieVersion,
```
Comment (drostron) [inline project/Dependencies.scala]: The other blocks align the 3rd column.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -96,12 +107,12 @@ object Dependencies {
   }
 
   val postgresql = Seq(
-    CommonDependencies.doobie.core     % "compile, test",
-    CommonDependencies.doobie.postgres % "compile, test"
+    "org.tpolecat" %% "doobie-core"     % doobieVersion % "compile, test",
```
Comment (drostron) [inline project/Dependencies.scala]: Same

**Comment by jedesah** [comment]
@drostron Any idea where scala `2.11.11` is coming in from?

**Comment by jedesah** [comment]
(in the first build failure)

**Comment by drostron** [comment]
@jedesah Strange, it even looks like `2.11.8` is being set for that stage.

**Comment by jedesah** [comment]
Reverted the upgrade to Spark as I have a suspicion it might be the cause of the build failure. (I guess I could have tested this locally before pushing...)

**Comment by drostron** [comment]
@jedesah Just a couple alignment nits and then this looks ready.

**Comment by jedesah** [comment]
@drostron done

**Pull request merged**